cmake/android/OpenCVDetectAndroidSDK.cmake

```diff
@@ -173,12 +173,12 @@ if(BUILD_ANDROID_PROJECTS)
   ocv_detect_android_sdk_tools()
   ocv_detect_android_sdk_build_tools()
 
-  if(ANDROID_SDK_TOOLS_VERSION VERSION_LESS 14)
+  if(DEFINED ANDROID_SDK_TOOLS_VERSION AND ANDROID_SDK_TOOLS_VERSION VERSION_LESS 14)
```

Comment:
Please clean CMake cache and ensure that configuration process doesn't stop on the line 64.

Reply:
Thanks, sorry for confusion. My sdk contained both `tools` and `cmdline-tools`. Needed reinstall.